text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/Context.java

```diff
@@ -42,10 +43,11 @@
    * Parses a MiniMessage string using all the settings of this context.
    *
    * @param message the message to parse
+   * @param resolvers additional tag resolvers
```

Comment:
```suggestion
   * @param resolvers additional tag resolvers, added to all other resolvers in this parse, but taking priority in the event of a name overlap
```

just to make it a bit more clear how this interacts with existing choices